client/angular/controllers/MainController.js

```diff
@@ -502,14 +478,6 @@
                     "width": tweetWidth + "px",
                 };
             }
-            return {
```

Comment:
How about having it always return the width without even checking for mobile? The reason it works fine now is because it's only ever called from a mobile template. I guess it should be renamed to reflect that - getMobileTweetWidth or something.